content/applications/marketing/events/lead_generation_rules.rst

```diff
@@ -0,0 +1,207 @@
+is clicked is revealed.
+
+The functionality of creating conditions for lead generation rules in this manner is similar to
+:ref:`how to filters are defined in the Marketing Automation application
+<marketing_automation/defining-filters>`.
+
+Click into the first field of the condition line to reveal a popover menu of alphabetically-ordered
+options to choose from. Once a selection (or selections) are made for the first field, click into
+the second field of the condition line to reveal a drop-down menu of corresponding modifiers. Then,
+in the third field select the final designating modifier to complete the custom condition line.
+
+Toggle the :guilabel:`Include archived` switch to also include archived attendee records in the
+condition.
+
+To the right of each condition line are three icons:
+
+- :icon:`fa-plus` :guilabel:`(plus sign) Add New Rule` icon: adds another rule to the condition.
+- :icon:`fa-sitemap` :guilabel:`(site map) Add branch` icon: adds a branch of two additional
+  condition lines that directly relate to the condition above it.
+- :icon:`fa-trash` :guilabel:`(trash can) Delete node` icon: deletes the condition line entirely.
+
+Users can see the number of attendee records that fall into that configured conditional rule(s) by
+looking at the :guilabel:`record(s)` located at the bottom of the :guilabel:`If The Attendees Meet
+These Conditions` section.
+
+Lead Default Values section
+---------------------------
+
+At the bottom of the :guilabel:`Lead Generation Rule` form, there is the :guilabel:`Lead Default
+Values` section.
+
+First, designate the :guilabel:`Lead Type` to be generated from this rule, either :guilabel:`Lead`
+or :guilabel:`Opportunity`.
+
+Then, if desired, choose to automatically assign the created leads to a specific sales team in the
+:guilabel:`Sales Team` field.
+
+Next, decide if the generated leads (or opportunities) should automatically be assigned to a
+specific salesperson in the :guilabel:`Salesperson` field.
+
+And, lastly, choose to automatically add specific tags to the created leads (or opportunities)
+generated from the rule. Multiple tags can be addeed, if desired.
+
+Sample rule: social media attendees
+===================================
+
+The following is a realistic example of how a company may choose to configure a lead generation rule
+for an event.
+
+In this sample, the company is looking to generate leads for every attendee who bought a *General
+Admission* ticket to a specific event (*Design Fair Los Angeles*), initially clicked a link for
+tickets from a social media post, and purchased their ticket online.
+
+First, the user would enter a memorable name related to the purpose of the rule in the
+:guilabel:`Rule Name` field, like `Social Media Attendees`. Then, select :guilabel:`Per Attendee` in
+the :guilabel:`Create` field, and :guilabel:`Attendees are created` in the :guilabel:`When` field.
```

Comment:
@ksc-odoo I haven't fully reviewed yet, but this part stuck out to me on quick look.

I'm not sure there's a relevant use case here to create a rule like this since `Attendees` by `Source` is more of a reporting filter which can be done right from the **Attendees** tab in the _Events_ app.

To give readers a working success model, consider covering how we internally set up LGRs for the Events team— the rules we have set up are pretty in-depth and really cover the functionality quite nicely ^^

Please remind me I can show you what I mean, if you like!